test/box_test.dart

```diff
@@ -114,7 +116,7 @@ void main() {
 
     box.put(TestEntity(tString: largeString));
 
-    items = box.getMany([1, 2]);
+    items = box.getMany([1, 2]) as List<TestEntity /*!*/ >;
```

Comment:
yeah, the cast is just for the `!`, otherwise, all the checks below working with those values would have to use `!` when accessing `items[i]`. This way the test throws directly on the commented line